### DISCUSSION TITLE
Quick Search addon

### DISCUSSION THREAD

**Discussion by RaptorDE** [Part-DB/Part-DB-server#507] [Show and tell]
In response to the Issue #305, I took a closer look at the search function in Project Bom and went down a ChatGPT rabbit hole. 😁 The result, a quick and dirty additional HTML page that provides a simple quick search. If anyone is interested, here is the [github](https://github.com/RaptorDE/part-db_quicksearch_addon).

![preview](https://github.com/Part-DB/Part-DB-server/assets/37591931/e0e25152-b763-4319-92bc-89ec69d2180d)

**Comment by RaptorDE** [comment]
I have improved the addon today and added new features.

New in version 1.2
* added Check for updates when the page is loaded (to disable it, comment out the line `window.onload = checkForUpdates;` at the end of the file)

New in version 1.1
* added icons and metadata for app shortcuts from Part-DB
* added links to Images
* added Footer with link to GitHub and Part-DB
* added Count of search results at the end of the table
* The URL to the Part-DB server no longer needs to be specified
* Automatic translation, based on the browser language
* English README.md

![Screenshot 2024-02-04 at 22-15-08 Quick Search(1)](https://github.com/Part-DB/Part-DB-server/assets/37591931/b9b31d63-4c18-4cb9-9cb1-f9f93df06f91)

**Comment by jbtronics** [comment]
Part-DB now offers a autocomplete / quicksearch for the navbar search field and the newly introduced search field on homepage:

![image](https://github.com/Part-DB/Part-DB-server/assets/5410681/8f5367d1-001a-4275-a3fe-1be9e5365761)


It is available in the demo instance: https://demo.part-db.de/en/
